Zones specs (#294)
PrestaShop/prestashop-specs: content/1.7/back-office/international/locations/zones/zones-listing.md

2. Show SQL query

After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.

3. Export to SQL Manager

After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager

# Bulk actions

Clicking on the _Bulk actions_ button allows performing bulk actions for the selected zones (using the checkboxes). A user can:

- Enable/Disable selection: these buttons will enable/disable all the zones that are selected.
- Delete selected: This button will delete the selected zone. 
After clicking on "Delete selected", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462)). The action can be canceled by clicking on the cross or on the cancel button.
After clicking on "Delete" button, the selected zone gets deleted, a successful message is displayed "The selection has been successfully deleted."

## Columns

1. ID

It's possible to sort by id asc or desc and search by id.

2. Zone

> It's possible to search and sort by a zone.

Reviewer: ```suggestion
It's possible to sort by zone asc or desc and search by zone.
```

---
title: Zones listing
weight: 1
---

# Zones listing

## Call to action

A call to actions is displayed in the header of this page:

 - Add new zone

👉  See the [specs of the add edit zone]({{<ref "add-edit-zone.md">}} "Add edit zone") 

## Settings wheel

When clicking on the settings wheel, the following options are displayed:

1. Refresh list

After clicking on it, the page is reloaded. Sorts and searches are kept.

2. Show SQL query

After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.

3. Export to SQL Manager

After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager

# Bulk actions

> Clicking on the _Bulk actions_ button allows performing bulk actions for the selected zones (using the checkboxes). A user can:

Reviewer 2: ```suggestion
The _Bulk actions_ button is active only if at least one row of the contact list is selected.Clicking on the _Bulk actions_ button allows performing bulk actions for the selected zones (using the checkboxes). A user can:
```

> - First checkbox allows to select/unselect all the zones in the list.

Reviewer: ```suggestion
Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
```

---
title: Zones listing
weight: 1
---

# Zones listing

## Call to action

A call to actions is displayed in the header of this page:

 - Add new zone

> 👉  See the [specs of the add new zone]({{<ref "add-new-zone.md">}} "Add new zone") 

Reviewer: After renaming the file, you could edit this link

## Settings wheel

When clicking on the settings wheel, the following options are displayed:

1. Refresh list

After clicking on it, the page is reloaded. Sorts and searches are kept.

2. Show SQL query

After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.

3. Export to SQL Manager

After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager

# Bulk actions

Clicking on the _Bulk actions_ button allows performing bulk actions for the selected zones (using the checkboxes). A user can:

> - Enable/Disable selection: these buttons will enable/disable all the checkboxes at the beginning of each row of the listing.

Reviewer: There is a mistake here, enable / disable allows to enable or disable the zone, not the checkboxes.
The select all / unselect all for the checkboxes is missing

> - Delete selected: This button will delete the selected zone. 

Reviewer: The part with the modale is missing, take a look at you did before for the states

## Columns

The page displays the zones list with the following columns:

1. ID

It's possible to sort by id asc or desc and search by id.

2. Zone

> It's possible to search by a zone.

Reviewer: And also to sort

3. Enabled

A drop-down allows to select: 

> - Yes

Reviewer: ```suggestion
```

> - No

Reviewer: ```suggestion
```

> When a search is performed with "yes" selected, only the zones that open in a new window are displayed.

Reviewer: I don't understand why you talk about a new window ?

Author: Not sure why I wrote new window, same one, just new view of only the searched zones:D

> When a search is performed with "no" selected, only the zones that not open in a new window are displayed.

Reviewer: I don't understand why you talk about a new window ?

4. Actions

**Edit.** This is the main action and it can be performed by clicking on the line or on the button. 

👉 See the [specs of the add new zone({{<ref "add-new-zone.md">}} "Add new zone") 

**Delete.** The zone can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
The action can be canceled by clicking on the cross or on the cancel button.
After clicking on "Delete" button, the zone is deleted and a successful message is displayed "Successful deletion."

> After clicking on "Delete selected", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462)). The action can be canceled by clicking on the cross or on the cancel button.

Reviewer: There is a duplicate content here

> After clicking on "Delete" button, the selected zone will be removed, and a successful message is displayed "The selection has been successfully deleted."

Reviewer: There is duplicate content here also
